### ISSUE TITLE
Using 1 ps eye camera?

### ISSUE THREAD

**Issue by armstrongdickpunch** [cboulay/PSMoveService#278]
Is it possible to use only one ps eye camera and 2 controllers and vridge for steam vr? Or do I need 2 cameras and 3 controllers? (Bc right now I don't really think I need position tracking bc I only have enough room to stand and play) and as of right now I only have 1 eye camera and in steam vr the controllers keep moving all around and behind me and what not and I followed the tutorial exactly. Any suggestions?

Also sorry if this isn't the right area to as questions I'm new to GitHub lol

**Comment by bclindner** [comment]
The extra cameras are to help pinpoint the position of the controllers and to get the controllers from different angles that might not be viewable from a single camera alone - behind you, for instance - so those will probably be needed if you want to keep your controls from jittering and moving around so much. However, I'm sure you *can* go without a third PS Move, technically - you would just need to run the SteamVR Room Setup and make your HMD default to about your standing height.

A poor calibration causes problems like you're describing on my side, so I'd suggest a re-calibration for tracker poses, gyro, magnetometer, etc. YMMV on this one; I've got 3 cameras and 3 moves so we might not have the same problems.

TL;DR: definitely go grab a second camera, it's worth it.

**Comment by HipsterSloth** [comment]
As bclinder mentioned, a second camera definately helps with tracking stability because it cam triangulate the controller position. Also make sure you have run the color calibration for each controller (not just one controller). Jittery tracking is often symptomatic of poor color filtering. 

Also for general troubleshooting report I recommend checking out the google group as the issues page is really meant for reproducable bugs and feature requests. Also you'll tend to get more active discussion there:

https://groups.google.com/forum/#!forum/psmoveservice
